doc/changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+Version 0.5.6
+-------------
+
+* Add again the ``pybench`` microbenchmark suite.
```

Comment:
The reverted benchmark was larger, I had to modify the patch. It should now be fixed.